src/adagrams.js

```diff
@@ -1,15 +1,181 @@
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const getRandomIdx = () => {
+  const NUM_OF_LETTERS_IN_LETTER_BANK = 96
+  return Math.floor(Math.random() * NUM_OF_LETTERS_IN_LETTER_BANK)
+};
+
+const getLetterBank = (LETTER_POOL)=> {
+  let letterBank = []
+  for(let letter in LETTER_POOL){
```

Comment:
Nitpick: spacing
```suggestion
  for (const letter in LETTER_POOL) {
```

`letter` is never reassigned in the for-loop so should be declared with `const` instead.